code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -709,7 +709,6 @@ Pass a positive integer as an argument to override a bot's default speed.
 	//No ..() to prevent strip panel showing up - Todo: make that saner
 	if(topic_denied(usr))
 		usr << "<span class='warning'>[src]'s interface is not responding!</span>"
-		href_list = list()

```

Comment:
Do this test this? *Extensively?* Please ensure that anyone locked out using this proc cannot access the bot specific functions.